### DISCUSSION TITLE
Issue Maintenance should count as "contributions"

### DISCUSSION THREAD

**Discussion by WesTyler** [community/community#22336] [New to GitHub]
<p>Currently the only Open Source maintenance that is “rewarded” with nice green squares on our contribution graphs are explicit code contributions (commits, PRs, reviews) or opening issues. But for some of us who stay busy keeping up the health of projects by organizing issues, applying labels and assignees, or by answering and closing issues none of this is reflected as “contributions”. </p>
<p>Is there any reason why these actions *couldn’t* count as contribututions in the future?</p>
<p>  - Add a label to an issue</p>
<p>  - Assign an issue to a maintainer</p>
<p>  - Comment and Close an issue</p>
<p>  - Have an issue assigned to you by another maintainer</p>

**Comment by Wabri** [comment]
<p>Because otherwise I would have 50 thousand contributions every month (not joking) and i dont want to mix the “real” code contributions with the “side” contributions. Maybe adding another type of contribution could be the solution you want.</p>

**Comment by mpboom** [comment]
<p>I do agree with <a class="mention" href="/westyler">@westyler</a> that it would be a good thing if the organizers of a community would be rewarded for their work. But I also do agree with <a class="mention" href="/wabri">@wabri</a> that things could get quite messy.</p>
<p>Since things like applying labels is something that is often done, it might be a good idea to consider contributions like PRs/commits/reviews a “full contribution”, while labelling and assigning issues could count as a “.1” contribution. 10 of these would make one full contribution. It’s interesting to think about.</p>

**Comment by WesTyler** [comment]
<p>I think that you both touched on the exact reason why I think this is an extremely valuable addition - our culture inherently places a significantly higher value on code contributions than anything else.</p>
<p>In my daily job, some of the hardest work that has the biggest payoff for our team is not the code that I write. It’s the help, mentoring, or training that I provide to my coworkers. In the same vein DevRels are commonly disregarded as “less important” than engineers, but in my experience our industry would collapse without them doing the hard people work.</p>
<p>As a more tangible example, the Node.js project maintainers are constantly triaging issues. Without those people managing the backlog of issues it would be nearly impossible as a code contributor or a user to find the information you need.</p>

**Comment by AndreaGriffiths11** [comment, marked as answer]
<p>Hi <a class="mention" href="/westyler">@westyler</a>, I really appreciate the time you’ve to take to write about this, and thanks for everyone for chiming in and keeping the discussion going. As someone whose, OSS contributions are not always coding 24/7 this topic is indeed personal. We have a system to relay feature requests internally to which I think this post would be valuable. Thanks for the feedback! I’ll take your suggestions and pass them along to the appropriate teams. </p>

**Comment by tomershafir** [comment]
any update? :)

**Comment by b8zhong** [comment]
Bumping for an update!

Motivation: merging a PR counts, so perhaps closing should do the same.

**Comment by shafik** [comment]
Issue triaging is a critical component for any large long existing open source project. Not rewarding this as a contribution feels like we are leaving out important members of our community.

Long running project live and die by their maintenance. It is an often underappreciated set of tasks. Screening bugs, getting them in front of the right eyes, tracking down regressions etc

Software Development Lifecycle, the life cycle includes all that other work.

**Reply by IsakBH** [reply]
Absolutely agree, bumping this. Open source would fall apart without issue triaging, and it should definitely be rewarded - even if it's only with a green pretty square on your profile.
